CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## 5.0.0
+
+* :exclamation: (improvement, **breaking**) `samples` is now a required field in `BasicOperationSchema` and friends. See [this doc](https://zapier.com/developer/documentation/v2/trigger-sample-results/) for more info ([PR](https://github.com/zapier/zapier-platform-schema/pull/32))
+* :exclamation: (improvement, **breaking**) `order` has been removed from `BasicDisplaySchema` ([PR](https://github.com/zapier/zapier-platform-schema/pull/27))
+
+* :tada: (new) The CLI can now run, test, and build on any version of Node.js! ([PR](https://github.com/zapier/zapier-platform-cli/pull/234))
+* :tada: (new) Read deploy key from env if available ([PR](https://github.com/zapier/zapier-platform-cli/pull/239))
+
+* (improvement) more specific error messages for invalid installs ([PR](https://github.com/zapier/zapier-platform-cli/pull/238))
```

Comment:
Maybe it's me being picky, but let's be consistent and capitalize the first letter in every sentence, i.e.,

> (improvement) **M**ore specific error messages for invalid installs